src/pages/pricing.tsx

```diff
@@ -223,22 +223,27 @@ const PricingPage: React.SFC<{}> = () => (
 
                     <section className="links">
                         <div>
-                            <h2>Open Source</h2>
+                            <h2>Unlimited Open Source</h2>
                             <p>
-                                Gitpod is free for up to 50 hours/month on any public repository. Just prefix any public GitHub or GitLab URL with <span className="text-with-galaxy-bg">gitpod.io/#</span> and start coding.
+                                Gitpod is free for up to 50 hours/month on any public repository. 
                             </p>
                             <p>
-                                If you're a professional open-source developer and think you need more, let us know, by applying for <strong>a free unlimited open source plan</strong>. Your GitHub / GitLab track record should clearly show your regular open-source contributions.
+                                If you're a professional open-source developer and need more hours, let us know, by applying for a <strong> free Unlimited Open Source Plan</strong>. Your GitHub / GitLab track record should clearly show your regular open-source contributions.
                             </p>
                             <a href="/contact/" className="btn">Apply Now</a>
                         </div>
                         <div>
                             <h2>For Students</h2>
                             <p>
-                                Students get the <strong>Unlimited plan for {isEurope() ? '€8' : '$9'}</strong> per month.
+                                Get the <strong> <a href="https://gitpod.io/subscription/"> Student Unlimited Plan </a>for {isEurope() ? '€8' : '$9'}</strong> per month.
```

Comment:
I think the word "plan" should not be part of the highlighted plan name (also, `target="_blank"` allows opening in a separate tab):

```suggestion
                                Get the <strong><a href="https://gitpod.io/subscription/" target="_blank">Student Unlimited</a> plan for {isEurope() ? '€8' : '$9'}</strong> per month.
```